README.md

```diff
@@ -136,51 +144,73 @@ Scanner vendors are supposed to implement Scanner Adapters exposing the Scanner
-   Status: 200 OK
-
-   {
-     "generated_at": "2019-08-07T12:17:21.854Z",
-     "artifact": {
-       "repository": "library/mongo",
-       "digest": "sha256:917f5b7f4bef1b35ee90f03033f33a81002511c1e0767fd44276d4bd9cd2fa8e"
-     },
-     "scanner": {
-       "name": "Microscanner",
-       "vendor": "Aqua Security",
-       "version": "3.0.5",
-     },
-     "severity": "High",
-     "vulnerabilities": [
-       {
-         "id": "CVE-2017-8283",
-         "package": "dpkg",
-         "version": "1.17.27",
-         "fix_version": "1.18.0",
-         "severity": "High",
-         "description": "...",
-         "links": [
-           "https://security-tracker.debian.org/tracker/CVE-2017-8283"
-         ]
-       },
-       ...
-     ]
-   }
-   ```
+   "vulnerabilities": [
+    {
+      "id": "CVE-2020-1967",
+      "package": "libcrypto1.1",
+      "version": "1.1.1c-r0",
+      "fix_version": "1.1.1g-r0",
+      "severity": "High",
+      "description": "Server or client applications that call the SSL_check_chain() function during or after a TLS 1.3 handshake may crash due to a NULL pointer dereference as a result of incorrect handling of the \"signature_algorithms_cert\" TLS extension. The crash occurs if an invalid or unrecognised signature algorithm is received from the peer. This could be exploited by a malicious peer in a Denial of Service attack. OpenSSL version 1.1.1d, 1.1.1e, and 1.1.1f are affected by this issue. This issue did not affect OpenSSL versions prior to 1.1.1d. Fixed in OpenSSL 1.1.1g (Affected 1.1.1d-1.1.1f).",
+      "links": [
+        "https://avd.aquasec.com/nvd/cve-2020-1967"
+      ],
+      "layer": {
+        "digest": "sha256:9d48c3bd43c520dc2784e868a780e976b207cbf493eaff8c6596eb871cbd9609",
+        "diff_id": "sha256:03901b4a2ea88eeaad62dbe59b072b28b6efa00491962b8741081c5df50c65e0"
+      },
+      "cvss": {
+        "nvd": {
+          "v2_vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
+          "v3_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
+          "v2_score": 5,
+          "v3_score": 7.5
+        },
+        "redhat": {
+          "v3_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
+          "v3_score": 7.5
+        }
+      },
+      "preferred_cvss": {
+        "score_v2": 5,
+        "score_v3": 7.5,
+        "vector_v2": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
+        "vector_v3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
+      },
+      "cwe_ids": [
+        "CWE-476"
+      ],
+      "vendor_attributes": {
+        "CVSS": {
+          "nvd": {
+            "V2Score": 5,
+            "V2Vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
+            "V3Score": 7.5,
+            "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
+          },
+          "redhat": {
+            "V3Score": 7.5,
+            "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
+          }
+        }
+      }
+    }
+    ...
+  ]
```

Comment:
Is this indentation okey?

Reply:
I have increased the indentation. Hopefully things will look Ok now